clients/src/test/java/org/apache/kafka/common/security/oauthbearer/internals/secured/ConfigurationUtilsTest.java

```diff
@@ -129,6 +140,33 @@ public void testFileWhitespace() {
         assertThrowsWithMessage(ConfigException.class, () -> testFile("    "), "must not contain only whitespace");
     }
 
+    @Test
+    public void testAllowedSaslOauthbearerUrlSystemProperty() {
```

Comment:
When we do unit tests we usually call the test method `test<METHOD_NAME>()`. Should this be called `testThrowIfURLIsNotAllowed()`?